src/main/java/spireMapOverhaul/zones/manasurge/ManaSurgeZone.java

```diff
@@ -125,39 +126,40 @@ public static List<AbstractManaSurgeModifier> getNegativeUncommonModifierList(bo
 
     public static void applyRandomTemporaryModifier(AbstractCard card) {
         if (AbstractDungeon.cardRandomRng.randomBoolean(COMMON_CHANCE)) {
-           List<AbstractManaSurgeModifier> commonModifierList = new ArrayList<>();
-           commonModifierList.addAll(getPositiveCommonModifierList(false));
-           commonModifierList.addAll(getNegativeCommonModifierList(false));
-            CardModifierManager.addModifier(card, Wiz.getRandomItem(commonModifierList, AbstractDungeon.cardRandomRng));
+            if (AbstractDungeon.cardRandomRng.randomBoolean(ENCHANT_CHANCE)) {
+                CardModifierManager.addModifier(card, Wiz.getRandomItem(getPositiveCommonModifierList(false), AbstractDungeon.cardRandomRng));
+            } else {
+                CardModifierManager.addModifier(card, Wiz.getRandomItem(getNegativeCommonModifierList(false), AbstractDungeon.cardRandomRng));
+            }
         } else {
-            List<AbstractManaSurgeModifier> uncommonModifierList = new ArrayList<>();
-            uncommonModifierList.addAll(getPositiveUncommonModifierList(false));
-            uncommonModifierList.addAll(getNegativeUncommonModifierList(false));
-            CardModifierManager.addModifier(card, Wiz.getRandomItem(uncommonModifierList, AbstractDungeon.cardRandomRng));
+            if (AbstractDungeon.cardRandomRng.randomBoolean(ENCHANT_CHANCE)) {
+                CardModifierManager.addModifier(card, Wiz.getRandomItem(getPositiveUncommonModifierList(false), AbstractDungeon.cardRandomRng));
+            } else {
+                CardModifierManager.addModifier(card, Wiz.getRandomItem(getNegativeUncommonModifierList(false), AbstractDungeon.cardRandomRng));
+            }
         }
     }
 
     public static void applyRandomPermanentModifier(AbstractCard card) {
         if (AbstractDungeon.cardRng.randomBoolean(COMMON_CHANCE)) {
-            List<AbstractManaSurgeModifier> commonModifierList = new ArrayList<>();
-            commonModifierList.addAll(getPositiveCommonModifierList(true));
-            commonModifierList.addAll(getNegativeCommonModifierList(true));
-            CardModifierManager.addModifier(card, Wiz.getRandomItem(commonModifierList, AbstractDungeon.cardRng));
-
+            if (AbstractDungeon.cardRandomRng.randomBoolean(ENCHANT_CHANCE)) {
```

Comment:
This should use cardRng since it's for a card reward, not just during combat. Other RNG uses in this method are already using cardRng.

```suggestion
            if (AbstractDungeon.cardRng.randomBoolean(ENCHANT_CHANCE)) {
```